CompanyManager.DAL/Entities/Person.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CompanyManager.DAL.Entities
+{
+    public class Person
+    {
+        public int Id { get; set; }
+
+        public int PostId { get; set; }
```

Comment:
Jako, że na razie chwilowo nie będziemy używać tego PostId to zakomentuj tą linijkę, powiązanie dodamy później, jak już się uda jakieś pierwsze dane wpisać do bazy danych.